radiant/dags/import_part.py

```diff
@@ -0,0 +1,224 @@
+    column_names = [desc[0] for desc in descriptions[0]]
+    dict_rows = [dict(zip(column_names, row, strict=False)) for row in results[0]]
+
+    cases = []
+    for case_id, grouped_rows in groupby(dict_rows, key=lambda x: x["case_id"]):
+        list_rows = list(grouped_rows)
+        case = Case(
+            case_id=case_id,
+            vcf_filepath=list_rows[0]["vcf_filepath"],
+            part=list_rows[0]["part"],
+            experiments=[
+                Experiment(
+                    seq_id=row["seq_id"],
+                    patient_id=row["patient_id"],
+                    sample_id=row["sample_id"],
+                    family_role=row["family_role"],
+                    sex=row["sex"],
+                    is_affected=row["is_affected"],
+                )
+                for row in list_rows
+            ],
+        )
+        cases.append(case.model_dump())
+
+    return [cases]
+
+
+dag_params = {
+    "part": Param(
+        default=None,
+        description="Integer that represents the part that need to be processed. ",
+    ),
+}
+
+std_submit_task_opts = SubmitTaskOptions(max_query_timeout=3600, poll_interval=10)
+
+
+@dag(
+    start_date=datetime.datetime(2021, 1, 1),
+    schedule=None,
+    catchup=False,
+    default_args=DEFAULT_ARGS,
+    tags=["radiant", "scheduled"],
+    dag_display_name="Radiant - Import for a partition",
+    dag_id=f"{NAMESPACE}-import-part",
+    params=dag_params,
+    render_template_as_native_obj=True,
+    doc_md=load_docs_md("import_part.md"),
+    template_searchpath=["/opt/airflow/dags/radiant/dags/sql"],
+)
+def import_part():
+    start = EmptyOperator(task_id="start", task_display_name="[Start]")
+
+    fetch_sequencing_experiment_delta = RadiantStarRocksOperator(
+        task_id="fetch_sequencing_experiment_delta",
+        sql="./sql/radiant/sequencing_experiment_select_delta.sql",
+        task_display_name="[StarRocks] Get Sequencing Experiments",
+        do_xcom_push=True,
+        output_processor=cases_output_processor,
+        parameters={"part": "{{ params.part }}"},
+    )
+
+    @task.short_circuit(task_id="sanity_check_cases", task_display_name="[PyOp] Sanity Check Cases")
+    def check_cases(cases: Any) -> Any:
+        return cases
+
+    cases = check_cases(fetch_sequencing_experiment_delta.output)
+
+    import_vcf = TriggerDagRunOperator(
+        task_id="import_vcf",
+        task_display_name="[DAG] Import VCF into Iceberg",
+        trigger_dag_id=f"{NAMESPACE}-import-vcf",
+        conf={"cases": cases},
+        reset_dag_run=True,
+        wait_for_completion=True,
+        poke_interval=2,
+    )
+
+    @task(task_id="extract_case_ids", task_display_name="[PyOp] Extract Case IDs")
+    def extract_case_ids(cases) -> dict[str, list[Any]]:
+        return {"case_ids": [c["case_id"] for c in cases]}
+
+    refresh_iceberg_tables = RadiantStarRocksOperator.partial(
```

Comment:
@jecos This is where we refresh the tables. For now, only `germline_*` tables are refreshed, I comment the Open Data ones, otherwise the task fails (and the downstream too)